docs/source/installation.rst

```diff
@@ -52,24 +53,48 @@ Option 2: Installing from source
     pip install torch-scatter
     pip install .
 
-.. Option 3: Installing via Docker
-.. -------------------------------
+Option 3: Installing via Docker
+-------------------------------
+ 
+Chemprop can also be installed with Docker, making it possible to isolate the Chemprop code and environment.
+To install and run Chemprop in a Docker container, first install Docker from docker_.
+You may then either ``pull`` and use official Chemprop images or ``build`` the image yourself.
 
-.. Chemprop can also be installed with Docker, making it possible to isolate the Chemprop code and environment. To install and run our code in a Docker container, first install docker from docker_. Then, run the following commands:
+.. note:: 
+    The Chemprop Dockerfile runs only on CPU and does not support GPU acceleration.
+    Linux users with NVIDIA GPUs may install the `nvidia-container-toolkit`_ from NVIDIA and modify the installation instructions in the Dockerfile to install versions of `torch` and `torch-scatter` which are compatible with your system's GPUs and drivers.
+    Adding the ``--gpus all`` argument to ``docker run`` will then allow Chemprop to run on GPU from within the container.
+    Users on other systems should install Chemprop from PyPI or source.
+
+.. _docker: https://www.docker.com/get-started/
+.. _`nvidia-container-toolkit`: https://docs.nvidia.com/datacenter/cloud-native/container-toolkit/latest/install-guide.html
```

Comment:
@kevingreenman if you want to capitalize the link to Docker I think you will have to do it like this

Reply:
Specifically line 70 in this comment